assembl/static2/js/app/pages/idea.jsx

```diff
@@ -102,6 +134,7 @@ class Idea extends React.Component {
                     contentLocale={contentLocale}
                     lang={lang}
                     data={topPosts}
+                    initialRow={this.topPostsToLoad(topPosts, ideaWithPostsData.idea.posts.edges)}
```

Comment:
I find `topPostsToLoad` name confusing, as it returns the index of the last top post that we want to load before the scroll (if I'm right). Maybe `getInitialRowIndex` or `getTopPostToScrollIndex` (hum...)

Reply:
ok, getInitialRowIndex it is